java/src/main/java/com/xxAMIDOxx/xxSTACKSxx/service/impl/MenuServiceImpl.java

```diff
@@ -15,38 +14,72 @@
 import java.util.Optional;
 import java.util.UUID;
 
+import static com.xxAMIDOxx.xxSTACKSxx.utils.UtilityMethods.pageRequestWithSort;
+
 @Service
 public class MenuServiceImpl implements MenuService {
 
+    private static final String RESTAURANT_ID = "restaurantId";
+    private static final String NAME = "name";
     private static Logger logger = LoggerFactory.getLogger(MenuServiceImpl.class);
 
-    private final MenuRepository menuRepository;
+    private MenuRepository menuRepository;
 
     public MenuServiceImpl(MenuRepository menuRepository) {
         this.menuRepository = menuRepository;
     }
 
     public List<Menu> all(int pageNumber, int pageSize) {
-
         int currentPage = 0;
-        final Sort sort = Sort.by(Sort.Direction.ASC, "Name");
+        //TODO:  Validate all UUID's in DB and change "Name" back to "name"
 
-        final CosmosPageRequest pageRequest = new CosmosPageRequest(
-                currentPage, pageSize, null, sort);
-
-        Page<Menu> page = this.menuRepository.findAll(pageRequest);
+        Page<Menu> page = menuRepository.findAll(pageRequestWithSort(Sort.Direction.ASC, "Name", currentPage, pageSize));
         logger.debug("Total Records: {}", page.getTotalElements());
         logger.debug("Total Pages: {}", page.getTotalPages());
 
         while (currentPage < pageNumber && page.hasNext()) {
```

Comment:
this pagination code needs to be used in all service operations. It'll not paginate otherwise

Reply:
Will be too much boilerplate. Going to contain pagination to the findAll call only.